test/test_torch.py

```diff
@@ -1208,6 +1208,26 @@ def test_dtype_out_match(self):
         d = torch.autograd.Variable(torch.DoubleTensor(2, 3))
         self.assertRaises(RuntimeError, lambda: torch.zeros((2, 3), out=d, dtype=torch.float32))
 
+    def test_make_subclass(self):
+        class SubTensor(torch.Tensor):
+            pass
+
+        t0 = torch.Tensor(0)
+        t1 = torch.Tensor([1, 2])
+        t2 = torch.Tensor([[3, 4], [5, 6]])
```

Comment:
nit: for tests, try to use the new style constructor `torch.tensor` rather than the legacy constructor